app/javascript/application.js

```diff
@@ -52,6 +52,27 @@ import "./tooltip";
 import "./truncate";
 import "./update-hidden-inputs-by-checkbox";
 
+// Patched this method so that scripts returned from the ajax call are executed.
+// Specifically the script for recaptcha
+Blacklight.modal.receiveAjax = function (contents) {
+  // does it have a data- selector for container?
+  // important we don't execute script tags, we shouldn't.
```

Comment:
```suggestion
```
Can we remove this one comment to avoid confusion?

Reply:
This is great and works perfectly for the recaptcha issue. This jquery-less version would strip other attributes from script tags (e.g., integrity, crossorigin) I believe? Not a problem here, but do you think it could be a concern in more general use?